spring-cloud-data-rest/src/main/java/org/springframework/cloud/data/rest/resource/StreamDefinitionResource.java

```diff
@@ -24,29 +24,37 @@
 import org.springframework.hateoas.mvc.ResourceAssemblerSupport;
 
 /**
- * todo: add javadoc
- *
- * todo: document that this is not thread safe
+ * A HATEOAS representation of a {@link StreamDefinition}.
+ * This class also includes a description of the
+ * {@link #status stream status}.
+ * <p>
+ * Note: this implementation is not thread safe.
  *
  * @author Patrick Peralta
  */
 @XmlRootElement
```

Comment:
Since we assume the controller always produces `application\json`, we may not need this annotation anymore.